# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/test/crashtests/crashtests.list
# Commit: 6b9661183d82
# Full Hash: 6b9661183d828f8c14e72a4ece61b5bc60ab166d
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-08-01 16:06:52
# Description:
#   Bug 1845350 - Prevent a division by zero when adjusting dts in MoofParser.cpp. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D184683
# ==============================================================================

diff -r 93e2ed328b0c -r 6b9661183d82 dom/media/test/crashtests/crashtests.list
--- a/dom/media/test/crashtests/crashtests.list	Tue Aug 01 07:07:08 2023 +0000
+++ b/dom/media/test/crashtests/crashtests.list	Tue Aug 01 07:07:30 2023 +0000
@@ -165,3 +165,4 @@
 load 1833894.mp4
 load 1835164.html
 load 1840002.webm
+load 1845350.mp4
